Task2/index2.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Gourmet au Catering</title>
+    <link rel="stylesheet" href="style2.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="brand">
+                <h1>Gourmet au Catering</h1>
+            </div>
+            <div class="nav">
+                <ul>
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#menu">Menu</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <div class="firstsec">
+        <div class="main image">
+            <img src="images\image1.jpg" alt="Le catering">
```

Comment:
Please take care of all the images path. Make sure you name the path properly. In the images src path images\image1 and folder name is Images.